src/test/java/apicurito/tests/TestRunner.java

```diff
@@ -43,7 +44,7 @@ public static void beforeTests() {
             if (!createdProject) {
                 ApicuritoTemplate.cleanNamespace();
```

Comment:
Also, change here `Template` to `Install` it works that way because of inheritance but it doesn't make sense from a logical point of view.